src/main/webapp/app/overview/exercise-details/course-exercise-details.module.ts

```diff
@@ -40,7 +43,29 @@ const routes: Routes = [
             {
                 path: '',
                 pathMatch: 'full',
-                loadChildren: () => import('../discussion-section/discussion-section.module').then((m) => m.DiscussionSectionModule),
+                component: !isOrion ? CourseExerciseDetailsComponent : OrionCourseExerciseDetailsComponent,
+                children: [
+                    {
+                        path: '',
+                        pathMatch: 'full',
+                        loadChildren: () => import('../discussion-section/discussion-section.module').then((m) => m.DiscussionSectionModule),
+                    },
+                ],
+            },
+            {
+                pathMatch: 'full',
+                path: 'problem-statement',
+                component: ProblemStatementComponent,
+            },
+            {
+                pathMatch: 'full',
+                path: 'problem-statement/:participationId',
+                component: ProblemStatementComponent,
+            },
+            {
+                path: 'feedback/:participationId/:resultId/:isTemplateStatusMissing',
```

Comment:
I don't understand this URL, it does not follow best practices.
Instead, it should be something like `participations/:participationId/results/:resultId/feedback?isTemplateStatusMissing=:isTemplateStatusMissing`

I.e. `isTemplateStatusMissing` should rather be a query param (not sure if this is possible)

It's also unclear why `isTemplateStatusMissing` is needed and where it comes from.

Reply:
Thanks for the suggestion. I have updated the route. Furthermore, I removed the `isTemplateStatusMissing` parameter as it is something that can be calculated by the web client itself.